app/models/schedule.rb

```diff
@@ -18,9 +18,7 @@ class Schedule < ActiveRecord::Base
                                              .pluck(:schedule_id)) }
 
   # == Validators ==
-  validates :root_task, presence: true,
-                        on: :update,
-                        unless: 'tasks.empty? || tasks.none?{ |t| !t.id.nil? }'
+  validates_presence_of :name
```

Comment:
Ahh, I was playing around with how to assign root task (trying to assign it so I could validate on submit in addition to update). I'll put that back though.